azure-pipelines.beta.yml

```diff
@@ -74,19 +74,19 @@ variables:
   - group: vh test users
   - group: Saucelabs
   - ${{ if containsValue(parameters.prodEnvs, parameters.environment) }}:
-    - group: vh-prod
+      - group: vh-prod
   - ${{ if not(containsValue(parameters.prodEnvs, parameters.environment)) }}:
-    - group: vh-preview
+      - group: vh-preview
 
 trigger: none
 pr: none
 schedules:
-- cron: "0 19 * * 0"
-  displayName: Weekly Beta build
-  always: true
-  branches:
-    include:
-    - master
+  - cron: '0 19 * * 0'
```

Comment:
A little concerned about this and the spacing changes in the yml files. Aren't they very spacing sensitive or am I mistaken? I will approve and let the pipelines determine that though.